packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -180,11 +180,38 @@ class _DomainCheckTable extends StatelessWidget {
                 children: <Widget>[
                   for (final error
                       in linkData.domainErrors.whereType<IosDomainError>())
-                    _IssuesBorderWrap(
-                      children: [
-                        _FailureDetails(errors: [error]),
-                      ],
-                    ),
+                    error == IosDomainError.existence
```

Comment:
the ternary check can be when defining the children of `_IssuesBorderWrap` since that is repeated:

```
_IssuesBorderWrap(
  children: error == IosDomainError.existence ? [...] : [...],
)
```